src/wagtail_localize_smartling/utils.py

```diff
@@ -21,19 +21,40 @@ def format_smartling_locale_id(locale_id: str) -> str:
     Format a locale ID for the Smartling API. Wagtail/Django use lower case
     (e.g. "en-us") whereas Smartling uses lower case for the language code and
     upper case for the region, if any (e.g. "en-US").
+
+    This behaviour is not applied if the REFORMAT_LANGUAGE_CODES setting is False
+
+    Also, this applies any mapping defined by the LOCALE_MAPPING_CALLBACK or
+    LOCALE_TO_SMARTLING_LOCALE settings.
     """
+    # Apply any mapping defined in settings
+    locale_id = smartling_settings.LOCALE_TO_SMARTLING_LOCALE.get(locale_id, locale_id)
+
+    if smartling_settings.REFORMAT_LANGUAGE_CODES is False:
+        return locale_id
+
+    # Reformat to match Smartling's format/casing
     original_parts = locale_id.split("-")
     if len(original_parts) == 1:
         return original_parts[0].lower()
     elif len(original_parts) == 2:
         return f"{original_parts[0].lower()}-{original_parts[1].upper()}"
+
     raise ValueError("Invalid locale ID")
 
 
 def format_wagtail_locale_id(locale_id: str) -> str:
     """
     The opposite of format_smartling_locale_id, return everything lower case.
+
+    This behaviour is not applied if the REFORMAT_LANGUAGE_CODES setting is False
     """
+    # Apply any mapping defined in settings
+    locale_id = smartling_settings.SMARTLING_LOCALE_TO_LOCALE.get(locale_id, locale_id)
+
+    if smartling_settings.REFORMAT_LANGUAGE_CODES is False:
```

Comment:
Pedantry: PEP-8 decrees that this should be `if not smartling_settings.REFORMAT_LANGUAGE_CODES:` and who am I to argue?

Reply:
True, true!

Reply:
@bcdickinson To avoid a clash on this branch, do you want me to fixup the two points you've just made, or are yuo happy to while in the area? I'm happy either way

Reply:
I've fixed these in my last commit, a mini re-review would be lovely!